### ISSUE TITLE
JunkYard

### ISSUE THREAD

**Issue by ABroHaam** [New-Chance-RP/Feedback#706]
<h2>Feature you are requesting.</h2>

JunkYard feature for unwanted unsellable cars.

This feature would allow people to sell vehicles to a junkyard for say 50% of the import value of the vehicle. This would make it easier for people to get rid of unwanted cars that may be very difficult to sell. They would take a big loss on the vehicle but would retain some of the money used to purchase the car. Mods or other changes to the vehicle after purchase I dont think should be included.

**Comment by brew4848** [comment]
I agree. I feel like no one else buys cars, else it is a nice import no one really buys it else you sell it for 20k+ below the stock price. So at that point, you might as well sell to the junkyard. especially when this takes 3+ weeks just to sell it.

**Comment by Zopfco** [comment]
> I agree. I feel like no one else buys cars, else it is a nice import no one really buys it else you sell it for 20k+ below the stock price. So at that point, you might as well sell to the junkyard. especially when this takes 3+ weeks just to sell it.

I think people tend to overvalue used vehicles.  If it's been heavily modified maybe you can get stock price or even a little over but the average used car would normally go for closer to 50-75% new cost, at best.  Engine etc mods add value but the cosmetic stuff tends to actually reduce it for me because I'm probably going to want to change those to suit me.

A local scrap yard should be just that - a place to scrap a car that nobody wants & still get a little money out of it, _**not**_  a way to bypass the used market because we don't want to negotiate a reasonable price.  If we're looking for a quick sale I think the dealerships are a better option but I understand they aren't always available.  

tl;dr - Scrapping should be a last option at rock-bottom pricing so it doesn't bypass the used market.

**Comment by KermitTheSlav** [comment]
Thats true but its alot harder to force the market to change below that, and helps the market by adding a price floor for each vehicle. Yes used cars sell for alot less in head but that doesnt apply to the city because in city cars dont randomly break down and cost thousands to repair and weeks of waiting, constant maintenence, lessening performance over time, cars in city last forever. If cars never broke down and lasted forever in the real world, they would not depreciate in that way, which is why its a feature needed in the city Example, the Elegy, sells new for 42,000 but it'll take a few weeks to sell it for 20k.

**Comment by ZeroInsomniac** [comment]
This issue is already on the Development list, and will be at a very very low percentage return, we want to encourage RP between players instead of having a "dump" for vehicles with no RP interaction.

**Comment by Grim-Reaper12** [comment]
@ZeroInsomniac Please have the price shown before you do sell the vehicles at a scrap yard or local vehicle sell location

**Comment by Senlarus** [comment]
Looking at adding this soon.
